src/features/channels/components/chat-content/message-reply/MessageReply.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Typography } from '@mui/material';
+import React from 'react';
+import useStyles from '../message/Message.styles';
+
+type props = {
+  name: string;
+  message: string;
+  date?: string;
+};
+
+const MessageReply: React.FC<props> = ({ name, message, date }) => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.reply}>
```

Comment:
This component looks very similar to the one for the Main message. Can't we reuse the same base so we don't repeat code?